<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetricsWindow</class>
 <widget class="QMainWindow" name="MetricsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>227</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>531</width>
    <height>227</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>531</width>
    <height>227</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Metrics</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/1024px-GNOME_Calculator_icon_2018.svg.png</normaloff>:/images/1024px-GNOME_Calculator_icon_2018.svg.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="inputLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Ubuntu&quot;;
background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Ubuntu&quot;;
background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);</string>
    </property>
    <item>
     <property name="text">
      <string>Kilometres</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Feets</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Miles</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="OutputLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Ubuntu&quot;;
background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="CheckBMIButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>140</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string>Check!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxOutPut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Ubuntu&quot;;
background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);</string>
    </property>
    <item>
     <property name="text">
      <string>Kilometres</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Feets</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Miles</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ExitButton</sender>
   <signal>clicked()</signal>
   <receiver>MetricsWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
